examples/mnist-classification/project.clj

```diff
@@ -0,0 +1,20 @@
+(defproject mnist-classification "0.9.4-SNAPSHOT"
+  :description "An example of using experiment/classification on mnist."
+  :dependencies [[org.clojure/clojure "1.8.0"]
+                 [thinktopic/experiment "0.9.4-SNAPSHOT"]
```

Comment:
@cnuernber --- how does Charles' comment here relate to your script that changes all the version numbers together?